onvm/onvm_nflib/onvm_common.h

```diff
@@ -311,6 +319,7 @@ struct onvm_service_chain {
 #define NF_NO_DEDICATED_CORES 10  // There is no space for a dedicated core
 #define NF_CORE_OUT_OF_RANGE 11   // The manually selected core is out of range
 #define NF_CORE_BUSY 12           // The manually selected core is busy
+#define NF_WAITING_FOR_MGR 13     // NF is waiting for a LPM request to be fulfilled
```

Comment:
NF_WAITING_FOR_LPM, waiting for mgr is too generic

Reply:
+1, please rename so it's more descriptive